src/NServiceBus.Core/Receiving/ReceiveAddresses.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace NServiceBus
+{
+    using System;
+    using Features;
+
+    //TODO should this be an interface?
+    /// <summary>
+    /// Lists the transport addresses this endpoint is consuming messages from.
+    /// </summary>
+    public class ReceiveAddresses
```

Comment:
Reading the comment above I wonder if `TransportAddresses` would be a better name?